README.md

```diff
@@ -215,8 +215,6 @@ This is an example command to run the local proxy in destination mode, on a tunn
 
 We recommend starting the destination application or server before starting the destination local proxy to ensure that when the local proxy attempts to connect to the destination port, it will succeed. When the local proxy starts in destination mode, it will first connect to the service, and then begin listening for a new connection request over the tunnel. Upon receiving a request, it will attempt to connect to the configured destination address and port. If successful, it will transmit data between the TCP connection and tunnel bi-directionally. 
 
-If a new instance of destination local proxy starts,  using the same client access token as a existing local proxy that is already connected, the old local proxy will be disconnected. Tunnel connection will be reset and the new tunnel connection will be established with the new instance of the destination local proxy. 
```

Comment:
what will happen now in this scenario? I am guessing user will get invalid token error on second instance of local proxy

Reply:
you can no longer re-use the token, so this scenario can no longer happen.